tests/kernel/run_ltp.pm

```diff
@@ -277,6 +278,12 @@ sub run {
         my $fin_msg    = "### TEST $test->{name} COMPLETE >>> ";
         my $cmd_text   = qq($test->{command}; echo "$fin_msg\$?");
         my $start_time = thetime();
+        my $set_rhost  = $is_network && $test->{command} =~ m/^finger01|ftp01|rcp01|rdist01|rsh01/;
+
+        if ($set_rhost) {
+            assert_script_run(q(export RHOST='localhost'));
```

Comment:
I hope as well. This is caused by combination of mrsh as rsh server and old tests requiring rsh. Best would be if LTP have converted all networking tests into new API.
I might find a way in the future to get rid of this.